packages/sources/svelte/src/components/navigation/VtmnSearch/enums.js

```diff
@@ -0,0 +1,11 @@
+export const VARIANTS = {
```

Comment:
A new naming has been implemented since yesterday on const -> VTMN_{component}_{enum name}
`VTMN_SEARCH_VARIANTS`

Reply:
Without the "S" at the end :)